deploy/ray-operator/controllers/common/constant.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+const (
+	// use as pod type decision
+	Head = "head"
+
+	// use as label key
+	rayclusterComponent = "raycluster.component"
+	rayIoComponent      = "rayclusters.ray.io/component-name"
+	rayOperator         = "ray-operator"
+	RayClusterOwnerKey  = "raycluster.instance.name"
+	ClusterPodType      = "raycluster.pod.type"
+
+	// use as separator
```

Comment:
separator for what?

Reply:
separator for name, here the podName will be rayclsuter-groupName-type-0, for example, raycluster-small-size-worker-0